ThePhasesOfCraftship/2_best_practice_first/strategicDesignPart1/exercises/1_RefactoringTo4Tiers/begin/src/bootstrap.ts

```diff
@@ -0,0 +1,40 @@
+import express from "express";
+import { Database } from "./database";
+import { PrismaClient } from "@prisma/client";
+import { StudentService } from "./services/studentService";
+import { ClassService } from "./services/classStervice";
+import { AssignmentService } from "./services/assignmentService";
+import StudentController from "./controllers/studentController";
+import { errorHandler } from "./shared/errorHandling";
+import ClasseController from "./controllers/classController";
+import AssignmentController from "./controllers/assignmentController";
+import Server from "./server";
+
+const cors = require("cors");
+const app = express();
+
+app.use(express.json());
+app.use(cors());
```

Comment:
app is also created as part of code in `server.ts` - so this looks like some leftover